### DISCUSSION TITLE
phantom notifications

### DISCUSSION THREAD

**Discussion by jbockle** [community/community#35381] [Other Feature Feedback, Questions, & Ideas] [answered]
my notifications aren't clearing,  it shows 6 but nothing there!

![image](https://user-images.githubusercontent.com/6231457/194443392-9fae376d-dca3-4ce6-9622-49e8621ea984.png)

**Comment by radar-blip** [comment, marked as answer]
Thats the old Ghost Notifications bug. To fix it, go through all your notifications in your inbox and done folders, select all and mark as read.
Its just a matter of marking all notifications as read. No matter how you do it.

**Reply by jbockle** [reply]
Thanks for the reply - thats not working for me though, select all doesn't select anything so marking read isn't available.  I tried moving done to inbox and marking read and the same 6 unread count is there.

**Reply by radar-blip** [reply]
Ok. Maybe something in this thread will help:  #6874
I didn't read through all of them but it appears some people have had some luck there.

Or here: #32712

**Reply by jbockle** [reply]
thanks! was able to resolve it with suggestions in the first link

**Reply by radar-blip** [reply]
Awesome. Glad one of those suggestions worked!
And glad I could help!

**Comment by justinmchase** [comment]
I have the same issue but the notification is in a repo that I lost access to or was deleted or something. I am unable to actually clear the notification.
